daemon/graphdriver/overlay/overlay.go

```diff
@@ -195,10 +211,12 @@ func (d *Driver) GetMetadata(id string) (map[string]string, error) {
 	return metadata, nil
 }
 
+//Cleanup simply returns nil and do not change the existing filesystem.
 func (d *Driver) Cleanup() error {
 	return nil
 }
 
+// Create creates the require root and other directories such as merge directory, etc based on the parent.
```

Comment:
s/require/required.
Let's get specific about the implementation too.
`Create created the upper, lower, and merge directories required for overlay fs. The parent is used for the lower.` (make sure this matches with the implementation, I didn't read it fully)